gaia-header.js

```diff
@@ -1,29 +1,52 @@
 ;(function(define){'use strict';define(function(require,exports,module){
+/*jshint laxbreak:true*/
 /*jshint esnext:true*/
+/*jshint boss:true*/
 
 /**
  * Dependencies
  */
 
-var Component = require('gaia-component');
-var fontFit = require('./lib/font-fit');
+var component = require('gaia-component');
+var fontFit = require('font-fit');
 
-// Load 'gaia-icons' font-family
+/**
+ * Load 'gaia-icons' font-family
+ */
 require('gaia-icons');
 
 /**
+ * Simple logger (toggle 0)
+ *
+ * @type {Function}
+ */
+var debug = 0 ? console.log.bind(console) : function() {};
+var map = [].map;
+
+/**
  * Supported action types
  *
  * @type {Object}
  */
-var actionTypes = { menu: 1, back: 1, close: 1 };
+const KNOWN_ACTIONS = {
```

Comment:
if you're serious about this, you might want to use Object.freeze?
(really up to you, I don't really care)